.env.example

```
@@ -21,11 +18,9 @@ MIDAZ_TRANSACTION_URL="http://localhost:3002/v1"
 MIDAZ_TIMEOUT=30                  # Timeout in seconds for API requests (default: 60)
 MIDAZ_DEBUG=false                  # Debug mode for verbose output (default: false)
 
-# Retry configuration
+# Retry configuration read by config.FromEnvironment
 MIDAZ_MAX_RETRIES=3               # Maximum retry attempts for failed requests (default: 3)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)